mffpy/tests/test_writer.py

```diff
@@ -86,6 +86,67 @@ def test_writer_writes():
         Clean-up failed of '{dirname}'.  Were additional files written?""")
 
 
+def test_writer_writes_multple_bins():
+    dirname = 'multiple_bins.mff'
```

Comment:
Could write this to `cache_dir` just in case the cleanup fails for some reason and we have a random MFF lying around.